src/arch/host/CMakeLists.txt

```diff
@@ -6,3 +6,5 @@ target_include_directories(sof_public_headers INTERFACE ${PROJECT_SOURCE_DIR}/sr
 
 # C & ASM flags
 target_compile_options(sof_options INTERFACE -g -O3 -Wall -Werror -Wl,-EL -Wmissing-prototypes -Wimplicit-fallthrough=3)
+
+add_subdirectory(lib)
```

Comment:
Please add empty newlines at end. I think checkpatch complains about it in C files but doesn't care about CMake.

Reply:
done